.prettierignore

```diff
@@ -0,0 +1,2 @@
+/working-group/
+/rfcs/
```

Comment:
Why shouldn't those be formatted?

Reply:
Generally I find it's easier if the things that people like to edit through GitHub are not formatted via prettier (because CI failing on them is annoying) but the main spec text itself should be nicely formatted. That, and the size of the changes was _massive_ for these folders!

Reply:
I'm definitely open to formatting everything with prettier though; we can slap a `<!-- prettier-ignore -->` in front of the WG attendees lists so it doesn't become a pain.

Reply:
I believe RFCs should not be formatted.  They are written as a way of communicating and should not be altered by anyone other than the original poster.

Reply:
I agree, perhaps a short code comment would be good to document that thinking.

Reply:
Comments added